.travis.yml

```diff
@@ -11,7 +11,7 @@ before_install:
 install:
   - pip install --upgrade pip
   - pip install --ignore-installed -r requirements_dev.txt
-  - pip install matplotlib==1.3.1
+  - pip install matplotlib==1.4.1
```

Comment:
Ok, we need to check if all the plotting functionality is compatible with this version, but if no tests are failing that's a good start. Good to know.